app/controllers/CampaignApi.scala

```diff
@@ -87,17 +87,22 @@ class CampaignApi(components: ControllerComponents, authAction: AuthAction[AnyCo
     result getOrElse InternalServerError
   }
 
-  def getCampaignPageViewsFromDatalake(id: String) = authAction { _ =>
-    CampaignService.getPageViews(id) match {
+  def getCampaignPageViewsFromDatalake(id: String) = authAction { implicit request =>
+    val territory = Territory(request.getQueryString("territory") getOrElse "global")
```

Comment:
Could add this as an optional param to the endpoint

Reply:
Yes - will change.